uui-components/src/layout/ControlGroup.module.scss

```diff
@@ -8,4 +8,9 @@
             box-shadow: none;
         }
     }
+
+    .uui-focus, &>:focus-within, &>:hover {
```

Comment:
Probably we need to create here separate stacking context, that this z-index will not affect other parts, for example modals with z-index: 0. For example, we can make it via `isolation: isolate;`.

Reply:
You're right 👍, I checked this case, a separate stacking context is needed. Thank you!
Done.